src/main/groovy/com/getkeepsafe/dexcount/DexMethodCountTask.groovy

```diff
@@ -34,7 +34,10 @@ class DexMethodCountTask extends DefaultTask {
     def BaseVariantOutput apkOrDex
 
     @OutputFile
-    def File outputFile
+    def File outputFileTxt
+
+    @OutputFile
```

Comment:
Let's avoid specifying multiple outputs here - the CSV file will only be generated conditionally, unlike `outputFile`.